ingestion/src/metadata/sampler/partition.py

```diff
@@ -163,6 +163,7 @@ def _handle_bigquery_partition(
                 partitionIntegerRangeStart=1,
                 partitionIntegerRangeEnd=10000,
             )
+        # TODO: Allow External Hive Partitioning for profiler
```

Comment:
Out of curiosity how is this related to BigQuery?

Reply:
>BigQuery supports external tables that can source data from various storage options, including Amazon S3, Google Cloud Storage (GCS), and Azure Blob Storage.

but for those external tables to have partitioning, one can create those using Hive Partitioning 
ref: https://cloud.google.com/bigquery/docs/external-data-cloud-storage#create-external-table-partitioned
      
      CREATE EXTERNAL TABLE `PROJECT_ID.DATASET.EXTERNAL_TABLE_NAME`
      WITH PARTITION COLUMNS
      OPTIONS (
      format = 'SOURCE_FORMAT',
      uris = ['GCS_URIS'],
      hive_partition_uri_prefix = 'GCS_URI_SHARED_PREFIX',
      require_hive_partition_filter = BOOLEAN);